src/trix/models/block.coffee

```diff
@@ -137,13 +143,13 @@ class Trix.Block extends Trix.Object
     @attributes[depth]
 
   canBeGroupedWith: (otherBlock, depth) ->
-    attributes = @attributes
     otherAttributes = otherBlock.getAttributes()
-    if attributes[depth] is otherAttributes[depth]
-      if attributes[depth] in ["bullet", "number"] and otherAttributes[depth + 1] not in ["bulletList", "numberList"]
-        false
-      else
-        true
+    otherAttribute = otherAttributes[depth]
+    attribute = @attributes[depth]
+
+    attribute is otherAttribute and
+      not (Trix.config.blockAttributes[attribute].group is false and
```

Comment:
Now that we’ve introduced `getAllAttributeNames` and `getListAttributeNames`, it’s a little confusing that when we inquire about attributes, sometimes we access the config object directly and sometimes we go through a helper.

Do you think this spot (and the other spots in the code base where we access [`Trix.config.textAttributes`](https://github.com/basecamp/trix/search?utf8=✓&q=%22config.textAttributes%22&type=Code) and [`Trix.config.blockAttributes`](https://github.com/basecamp/trix/search?utf8=✓&q=%22config.blockAttributes%22&type=Code)) would benefit from the helper treatment?